games/ShootEmUp/scripts/Player.gd

```diff
@@ -19,7 +19,8 @@ var laser = preload("res://games/ShootEmUp/player_laser.tscn")
 @onready var animation_player = %AnimationPlayer
 @onready var hurt_audio = %HurtAudio
 @onready var death_audio = %DeathAudio
-#var death_audio_source: AudioStreamWAV
+@onready var impact_audio = %ImpactAudio
+@onready var explosion_death_audio = %DeathExplosionAudio
```

Comment:
This is by far not a big deal really, so you can leave it if you want, but I noticed the variable is called `explosion_death_audio`, while the node is `DeathExplosionAudio` with "explosion" and "death" swapped, maybe could make it consistent by changing one of them.